orderbook/src/api/get_fee_and_quote.rs

```diff
@@ -0,0 +1,238 @@
+use crate::fee::{MinFeeCalculating, MinFeeCalculationError};
+use anyhow::Result;
+use chrono::{DateTime, Utc};
+use ethcontract::{H160, U256};
+use model::h160_hexadecimal;
+use model::{order::OrderKind, u256_decimal};
+use serde::{Deserialize, Serialize};
+use shared::{
+    conversions::{big_int_to_u256, U256Ext},
+    price_estimate::{PriceEstimating, PriceEstimationError},
+};
+use std::convert::Infallible;
+use std::sync::Arc;
+use warp::{hyper::StatusCode, reply, Filter, Rejection, Reply};
+
+#[derive(Deserialize)]
+struct Query {
+    #[serde(with = "h160_hexadecimal")]
+    sell_token: H160,
+    #[serde(with = "h160_hexadecimal")]
+    buy_token: H160,
+    // For sell orders in sell token, for buy orders in the buy token.
+    #[serde(with = "u256_decimal")]
+    in_amount_before_fees: U256,
```

Comment:
I like the name. Although for buy orders, the name might be confusing, cause it's actually after fees :)

Reply:
I am starting to think we might want to routes to remove all ambiguity. One for selling and one for buying.

Reply:
If it's not too much overhead, it would be ideal yep.